mobile/src/main/java/ch/epfl/sdp/mobile/ui/game/GameScreenState.kt

```diff
@@ -9,7 +9,7 @@ import androidx.compose.runtime.Stable
  * @param Piece the type of the pieces of the underlying [ChessBoardState].
  */
 @Stable
-interface GameScreenState<Piece : ChessBoardState.Piece> : ChessBoardState<Piece> {
+interface GameScreenState<Pieces : ChessBoardState.Piece> : MovableChessBoardState<Pieces> {
```

Comment:
```suggestion
interface GameScreenState<Piece : ChessBoardState.Piece> : MovableChessBoardState<Piece> {
```

Reply:
Fixed in commit https://github.com/epfl-SDP/android/pull/240/commits/e937b3eef4a540d698d097c248e35b2ba29c6dd3